### ISSUE TITLE
Migration dependencies reference nonexistent parent node on AWS Elastic Beanstalk

### ISSUE THREAD

**Issue by marchss** [yourlabs/django-cities-light#100]
This is a mysterious issue bothering me. `django-admin makemigrations` and `django-admin migrate` on local dev machine works PERFECTLY.

**However, on AWS Elastic Beanstalk, both commands complains.**

My model is similar to 
```python
class Flavor(models.Model):
         name = models.CharField(max_length=100, blank=False, default=_('Sweet'))
         city = models.ForeignKey('cities_light.City', verbose_name=_('City'), help_text=_('City Foreign Key'))
```

In app icecream/migrations/0001_initial.py, it has
```python
class Migration(migrations.Migration):
    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('cities_light', '0004_auto_20151021_1129'),
    ]
    blabalbalbla.....
```

```python
Traceback (most recent call last):
  File "/opt/python/run/venv/bin/django-admin", line 11, in <module>
    sys.exit(execute_from_command_line())
  File "/opt/python/run/venv/local/lib/python3.4/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/opt/python/run/venv/local/lib/python3.4/site-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/python/run/venv/local/lib/python3.4/site-packages/django/core/management/base.py", line 390, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/python/run/venv/local/lib/python3.4/site-packages/django/core/management/base.py", line 441, in execute
    output = self.handle(*args, **options)
  File "/opt/python/run/venv/local/lib/python3.4/site-packages/django/core/management/commands/makemigrations.py", line 63, in handle
    loader = MigrationLoader(None, ignore_no_migrations=True)
  File "/opt/python/run/venv/local/lib/python3.4/site-packages/django/db/migrations/loader.py", line 47, in __init__
    self.build_graph()
  File "/opt/python/run/venv/local/lib/python3.4/site-packages/django/db/migrations/loader.py", line 294, in build_graph
    _reraise_missing_dependency(migration, parent, e)
  File "/opt/python/run/venv/local/lib/python3.4/site-packages/django/db/migrations/loader.py", line 264, in _reraise_missing_dependency
    raise exc
  File "/opt/python/run/venv/local/lib/python3.4/site-packages/django/db/migrations/loader.py", line 290, in build_graph
    self.graph.add_dependency(migration, key, parent)
  File "/opt/python/run/venv/local/lib/python3.4/site-packages/django/db/migrations/graph.py", line 116, in add_dependency
    parent
django.db.migrations.graph.NodeNotFoundError: Migration icecream.0001_initial dependencies reference nonexistent parent node ('cities_light', '0004_auto_20151021_1129')
```

In the ./lib/python3.4/site-packages/cities_light/migrations, it indeed has the 0004_auto_20151021_1129.py file.

Why does it still complains the **reference nonexistent parent node**? If I remove all database, migrations, and remigrate, the icecream.0001_initial migrations file won't depend on `('cities_light', '0004_auto_20151021_1129')`, but depend on some timestamp generated today
```
('cities_light', '0004_auto_20151212_1403'),
```

Obviously, this 0004_auto_20151212_1403.py does not exist in the ./lib/python3.4/site-packages/cities_light/migrations. But the django-admin makemigrations and migrate do not complain anything and the Django app is 'working' again.

**Comment by jpic** [comment]
Oops, i don't see the 0004 migration here: https://github.com/yourlabs/django-cities-light/tree/stable/3.x.x/cities_light/migrations

So, how did it get there in your repo ?

**Comment by jpic** [comment]
Apparently, this is a duplicate of #96, @st4lk what's your take on this ?

**Comment by jpic** [comment]
Could you guys please try PR #101 see if it fixes it ?

**Comment by marchss** [comment]
Thanks for the reply! Yes, it is a dupe of #96     I find a workaround to move the lib into the Django project, clear all previous migration files and remigrate so that everything is just inside 0001_initial.py

**Comment by jpic** [comment]
A fix has been released in 3.2.0.
